spec.html

```diff
@@ -40468,7 +40494,7 @@ <h1>WeakMap.prototype.set ( _key_, _value_ )</h1>
           1. Let _M_ be the *this* value.
           1. Perform ? RequireInternalSlot(_M_, [[WeakMapData]]).
           1. Let _entries_ be the List that is _M_.[[WeakMapData]].
-          1. If Type(_key_) is not Object, throw a *TypeError* exception.
+          1. If CanBeHeldWeakly(_key_) is *false*, throw a *TypeError* exception.
```

Comment:
personally i'd find it nice if the refactor of "type is object" to "canbeheldweakly" was done in a separate commit (or PR) - ie, that one first, and the "allow symbols" second. I think it would make the changes much easier to review.

not sure what @tc39/ecma262-editors think about that?

Reply:
During stage 2 I've been prioritizing keeping the git history intact, so hopefully easier for an individual to see what has changed since the last time they reviewed.

If this gets to stage 3 we could definitely look at splitting apart the editorial and normative changes into separate PRs as you suggest.